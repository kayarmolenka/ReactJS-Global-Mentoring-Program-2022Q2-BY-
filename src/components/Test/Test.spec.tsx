import { render } from '@testing-library/react';
import React from 'react';
import { Test } from './Test';

describe('Test', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<Test />);

    expect(asFragment()).toMatchSnapshot();
  });
});
