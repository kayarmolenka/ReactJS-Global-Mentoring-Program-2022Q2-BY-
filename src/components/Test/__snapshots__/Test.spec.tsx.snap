// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test should match snapshot 1`] = `
<DocumentFragment>
  <h1>
    Test
  </h1>
</DocumentFragment>
`;
